docs/source/modin/supported/dataframe_supported.rst

```diff
@@ -0,0 +1,472 @@
+| ``insert``                  | Y                               |                                  |                                                    |
++-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
+| ``interpolate``             | N                               |                                  |                                                    |
++-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
+| ``isetitem``                | N                               |                                  |                                                    |
++-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
+| ``isin``                    | Y                               |                                  |                                                    |
++-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
+| ``isna``                    | Y                               |                                  |                                                    |
++-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
+| ``isnull``                  | Y                               |                                  |                                                    |
++-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
+| ``items``                   | N                               |                                  |                                                    |
++-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
+| ``iterrows``                | Y                               |                                  |                                                    |
++-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
+| ``itertuples``              | Y                               |                                  |                                                    |
++-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
+| ``join``                    | P                               |                                  | ``D`` if given then ``validate`` param.            |
```

Comment:
should it be `given the validate param`?

Reply:
yeah, good catch!